integration_tests/udev/disks.go

```diff
@@ -19,7 +19,7 @@ package udev
 import (
 	"fmt"
 	"github.com/openebs/node-disk-manager/integration_tests/utils"
-	"math/rand"
+	"math/rand" #nosec G404
```

Comment:
This will give compile error. Need to replace with
```suggestion
	"math/rand" // #nosec G404
```